pnr/openroad-gui/condarc

```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```

Comment:
curious why conda-forge is needed?

Reply:
If I remember correctly, it was for qt.

Reply:
managed to drop the dependency on conda-forge by using cdt.